release-notes/opensearch-ml-common.release-notes-2.11.1.0.md

```diff
@@ -0,0 +1,27 @@
+## Version 2.11.0.0 Release Notes
+ 
+Compatible with OpenSearch 2.11.1
+ 
+ 
+### Enhancements
```

Comment:
Should we keep this part if no item?

Reply:
Isn't that same for other cases, like `Documentation`, `Maintenance`, `Refactoring`?

Reply:
Do you want me to remove other empty headers too?

Reply:
yes, I think we should remove all other empty headers too